### DISCUSSION TITLE
Investigate DAMS for museums & integration with Arctos

### DISCUSSION THREAD

**Discussion by AJLinn** [ArctosDB/arctos#4342] [Ideas]
From 2018-10-11 AWG:
DAMS system integration with Arctos - how do we compare already to DAMS specifications?

Some ideas:
https://www.capterra.com/digital-asset-management-software/ 
https://www.predictiveanalyticstoday.com/best-digital-asset-management-software/ 
https://www.museumsandtheweb.com/mw2012/papers/10_museums_12_months_1_dams_adventures_in_cent.html

**Comment by Jegelewicz** [comment]
@ArctosDB/media

**Comment by dustymc** [comment]
What is the functionality that we're looking for?

**Comment by AJLinn** [comment]
I think we're trying to determine what elements people like about traditional DAMS and if Arctos can act in the same capacity (and thus, this element can be part of our "marketing" language). When I submit grant proposals that have anything to do with digital media, reviewers go crazy about metadata, file storage, file formats, file naming conventions, and other things that I feel like I have a basic knowledge of and feel like Arctos is taking care of for me (when I upload media and assign relationships, descriptions, comments, etc). But reviewers are not getting the point as to why we're not using DAMs on top of the integration into Arctos.

I think Arctos (and by extension TACC) is capable of being our DAMS as well as our CMS. Maybe we all just need to read up on things (so far the best & most relevant info is in the Museums and the Web link above). This is the commercial system they are using at the Balboa Park Online Collaborative: http://www.piction.com/site/

I know I need/want to do more reading about this and have more discussions about what Arctos is doing, can do, and what we want it to do.

**Comment by dustymc** [comment]
Thanks!

I think it's safe to say that Arctos' model can support most anything - the Media node is completely normalized, it's hard to imagine any sort of data that won't fit. Going only from Balboa's budget, I suspect planning and building snazzy UIs (and whatever else is missing) would be cheaper than plugging in to an existing DAMS, and would result in a more integrated system that could do more STUFF from more perspectives for more users.

**Comment by Jegelewicz** [comment]
In some respects, I agree, but right now our "DAMS" capability is a little fuzzy and not quite what might be expected in a traditional DAMS. For instance, we don't really have a way right now to bring up a RAW media file along with all of its associated derivatives. In order for us to market our system as a DAMS or present it as such to a funding entity, I think we need to get a good template of DAMS functionality and compare it to what we have now, then decide if we should improve upon what we have or look for some kind of plug-in.

**Comment by Jegelewicz** [comment]
We can all do this in our massive amounts of free time... :-)

**Comment by dustymc** [comment]
>not quite what might be expected in a traditional DAMS.

Agreed - and that's not necessarily a bad thing.

>compare it to what we have now

Yes, that would be great - I certainly don't have a solid idea of what a DAMS is supposed to do or how it does it.

>don't really have a way right now to bring up a RAW media file along with all of its associated derivatives

Sure we do - http://arctos.database.museum/media/10208314 and http://arctos.database.museum/media/10281215 are both original-->derivatives viewpoints (although obviously only the image has actual RAW data). And we can do much more than that with the same mechanism - we can link media over time (http://arctos.database.museum/project/10000838, although I can't find the retakes) and etc. Media has access to the same stuff as anything else, so we can exploit projects, events, specimens, other Media, etc. to make or derive all sorts of relationships. (And not just *ogical relationships - a yearly-or-something photo of an item on display to monitor degradation might be neat.)

In any case, by "model" I meant "back end." I'm pretty happy to defend that - I don't think there's anything more powerful anywhere. The UIs are what I think will need improvement if we're to flip things around and treat Media as much more than specimen data. I think there are several ways we could get there, wherever "there" ends up being, anything from writing APIs for commercial products to just plugging in JS apps.

**Comment by Jegelewicz** [comment]
see #1785

**Comment by Jegelewicz** [comment]
Open Source - https://omeka.org/s/ not called a DAMS, but seems like it works like one.

**Comment by mkoo** [comment]
Omeka is more like wordpress for archives and databases as far as I know-- we have it installed on one of our servers if you want to try it. So not sure I see this as a DAMS for Arctos. Does work well with IIIF which is another reason we should move into that for Arctos media. If we have more archival compliant assets in Arctos we could explore talking to Omeka to build standalone websites-- a cool IMLS grant idea?!
